src/data/sponsors.json

```diff
@@ -126,16 +97,8 @@
           "en": "6 personalized posts on Facebook (3 each semester)"
```

Comment:
Change it to "(3 cada semestre)", line 96 please

Reply:
Also, line 93 and 97, please change the "(2 each month)" to "(two each month)" and "3" to "three", so there can be uniformity within the information.